src/vs/base/browser/mouseEvent.ts

```diff
@@ -128,7 +128,7 @@ export class StandardWheelEvent {
 	public readonly deltaX: number;
 	public readonly target: Node;
 
-	constructor(e: IMouseWheelEvent | null, deltaX: number = 0, deltaY: number = 0) {
+	constructor(targetWindow: Window, e: IMouseWheelEvent | null, deltaX: number = 0, deltaY: number = 0) {
```

Comment:
Maybe we can simplify the change if we get the window from the event itself:
```
const devicePixelRatio = e.view?.devicePixelRatio || 1;
```

Reply:
Addressed in https://github.com/microsoft/vscode/pull/200776/commits/6bffa641e77f5f3f313a746f953b5857c3538e0b